### PR TITLE
Init Azure.ResourceManager.FluidRelay

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/fluidrelay/Azure.ResourceManager.FluidRelay/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
```
Comment (archerzz) [inline sdk/fluidrelay/Azure.ResourceManager.FluidRelay/src/Properties/AssemblyInfo.cs:4]: See https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=sdk-generation-from-updated-spec
```
6. Update the following information in AssemblyInfo.cs
  - AssemblyFileVersion (Must for new nuget package)
  - AssemblyVersion
```

File: sdk/fluidrelay/Azure.ResourceManager.FluidRelay/api/Azure.ResourceManager.FluidRelay.netstandard2.0.cs [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource>> GetAsync(string fluidRelayContainerName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.FluidRelay.FluidRelayContainerResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class FluidRelayContainerData : Azure.ResourceManager.Models.ResourceData
+    {
+        public FluidRelayContainerData() { }
+        public string FrsContainerId { get { throw null; } }
```
Comment (archerzz) [inline sdk/fluidrelay/Azure.ResourceManager.FluidRelay/api/Azure.ResourceManager.FluidRelay.netstandard2.0.cs:19]: Are `FrsContainerId` and `FrsTenantId` GUID?

File: sdk/fluidrelay/Azure.ResourceManager.FluidRelay/src/Generated/Models/FluidRelayServerPatch.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        {
+            Tags = new ChangeTrackingDictionary<string, string>();
+        }
+
+        /// <summary> Resource tags. </summary>
+        public IDictionary<string, string> Tags { get; }
+        /// <summary> The type of identity used for the resource. </summary>
+        public ManagedServiceIdentity Identity { get; set; }
+        /// <summary> The geo-location where the resource lives. </summary>
+        public string Location { get; set; }
```
Comment (Yao725) [inline sdk/fluidrelay/Azure.ResourceManager.FluidRelay/src/Generated/Models/FluidRelayServerPatch.cs:28]: Is this location an `AzureLocation`?

File: sdk/fluidrelay/Azure.ResourceManager.FluidRelay/src/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/fluidrelay/Azure.ResourceManager.FluidRelay/src/Properties/AssemblyInfo.cs:4]: All track 2 and dataplane sdk don't have these properties

File: sdk/fluidrelay/Azure.ResourceManager.FluidRelay/api/Azure.ResourceManager.FluidRelay.netstandard2.0.cs (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/fluidrelay/Azure.ResourceManager.FluidRelay/api/Azure.ResourceManager.FluidRelay.netstandard2.0.cs:19]: Confirming with service team

Comment (ArthurMa1978) [reply sdk/fluidrelay/Azure.ResourceManager.FluidRelay/api/Azure.ResourceManager.FluidRelay.netstandard2.0.cs:19]: Confirmed it's `guid`

File: sdk/fluidrelay/Azure.ResourceManager.FluidRelay/src/Generated/FluidRelayContainerData.cs [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        /// <param name="provisioningState"> Provision states for FluidRelay RP. </param>
+        internal FluidRelayContainerData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, string frsTenantId, Guid? frsContainerId, ProvisioningState? provisioningState) : base(id, name, resourceType, systemData)
+        {
+            FrsTenantId = frsTenantId;
+            FrsContainerId = frsContainerId;
+            ProvisioningState = provisioningState;
+        }
+
+        /// <summary> The Fluid tenantId for this container. </summary>
+        public string FrsTenantId { get; }
```
Comment (Yao725) [inline sdk/fluidrelay/Azure.ResourceManager.FluidRelay/src/Generated/FluidRelayContainerData.cs:39]: Is `FrsTenantId` not a Guid?

**Review by Yao725 — APPROVED** [review]

**Pull request merged**